python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
No the example you give should be:
```
from Ganga.Core.GangaRepository import getRegistry
imported_jobs = getRegistry('jobs').getProxy()
assert jobs is imported_jobs
```
(This ```assert``` will only work on this PR as the proxy gets regenerated on every call before the changes I made here!)


The ```jobs``` exposed to the user at Runtime is NOT a registry object as you'd naively expect it's a ```JobRegistrySliceProxy``` around a ```JobRegistrySlice``` which has it's internal ```.objects``` store (list) replaced with the _jobs_ repository.